tests/publiccloud/storage_perf.pm

```diff
@@ -99,19 +100,20 @@ sub analyze_previous_series {
         my $previous_records_mean = get_mean_from_db($args, " limit 5 offset 5");
         # we do analysis only if we get back some non-zero values
         if (defined($previous_records_mean) && defined($last_records_mean) && $previous_records_mean != 0 && $last_records_mean != 0) {
-            record_info('ANALYZE', "Checking " . $current_load_type);
-            # this formula detect if mean values differs more than 10%
-            if ((abs($previous_records_mean - $last_records_mean) / $previous_records_mean) * 100 > 10) {
-                die sprintf("Anomaly occurred in http://openqa-perf.qa.suse.de/d/hoRc37HWz/storage-performance?orgId=1&var-os_flavor=%s&var-os_version=%s", $args->{os_flavor}, $args->{os_flavor});
+            my $diff_percents = (abs($previous_records_mean - $last_records_mean) / $previous_records_mean) * 100;
+            record_info('ANALYZE', sprintf("Checking %s:\nFormula: (abs(\$previous_records_mean - \$last_records_mean) / \$previous_records_mean) * 100 = (abs(%s - %s) / %s) * 100 = %s%", $current_load_type, $previous_records_mean, $last_records_mean, $previous_records_mean, $diff_percents));
+            # This detects if mean values differs more than 10%
+            if ($diff_percents > 10) {
+                record_soft_failure(sprintf("Anomaly occurred in http://openqa-perf.qa.suse.de/d/hoRc37HWz/storage-performance?orgId=1&var-os_flavor=%s&var-os_version=%s", $args->{os_flavor}, $args->{os_flavor}));
+                $return = 1;
+            } else {
+                record_info('PASS', sprintf("Anomaly occurred in http://openqa-perf.qa.suse.de/d/hoRc37HWz/storage-performance?orgId=1&var-os_flavor=%s&var-os_version=%s", $args->{os_flavor}, $args->{os_flavor}));
```

Comment:
I would like to rephrase this. The terms "previous mean" and "last mean" are not understandable unless you dig into the code.
```suggestion
                record_info('PASS', sprintf("For %s\nComparing average of dataset over 5 points\nReference dataset: %d \nCurrent dataset: %d", $current_load_type, $previous_records_mean, $last_records_mean));
```

WDYT? The aim is to make this understandable for people without the need to dive into the code.

Reply:
Is it 'dataset'? What about 'record'?

Reply:
generally I agree with rephrasing but I would keep URL to grafana ...

Reply:
"Comparing average of dataset over 5 points" - well without actually knowing what is going on just by this sentence I would not be able to understand something , but because my version is NOT better I am fine to keep yours :) 

"Reference dataset/Current dataset" - is more confusing and this part needs to be changed . It is not "dataset" per ce it is mean of it . So I think "Reference mean" is more accurate